json-rpc/types/src/views.rs

```diff
@@ -88,6 +88,8 @@ pub enum AccountRoleView {
         base_url_rotation_events_key: EventKey,
         preburn_queues: Option<Vec<PreburnQueueView>>,
     },
+    #[serde(rename = "treasury_compliance")]
+    TreasuryCompliance { diem_id_domain_events_key: EventKey },
```

Comment:
EventKey needs to be Option 
and add `#[serde(skip_serializing_if = "Option::is_none")]`